streams/src/test/java/org/apache/kafka/streams/perf/YahooBenchmark.java

```diff
@@ -136,51 +128,48 @@ private boolean maybeSetupPhaseEvents(final String topic, final String clientId,
         String[] eventTypes = new String[]{"view", "click", "purchase"};
         Random rand = new Random();
         // initialize topics
-        if (parent.loadPhase) {
-            if (skipIfAllTests) {
-                // if we run all tests, the produce test will have already loaded the data
-                if (parent.testName.equals(SimpleBenchmark.ALL_TESTS)) {
-                    // Skipping loading phase since previously loaded
-                    return true;
-                }
+        if (skipIfAllTests) {
+            // if we run all tests, the produce test will have already loaded the data
+            if (parent.testName.equals(SimpleBenchmark.ALL_TESTS)) {
+                // Skipping loading phase since previously loaded
+                return true;
             }
-            System.out.println("Initializing topic " + topic);
+        }
+        System.out.println("Initializing topic " + topic);
 
-            Properties props = new Properties();
-            props.put(ProducerConfig.CLIENT_ID_CONFIG, clientId);
-            props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, parent.kafka);
-            props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
-            props.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, ByteArraySerializer.class);
+        Properties props = new Properties();
+        props.put(ProducerConfig.CLIENT_ID_CONFIG, clientId);
+        props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, parent.kafka);
+        props.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
+        props.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, ByteArraySerializer.class);
 
-            KafkaProducer<String, byte[]> producer = new KafkaProducer<>(props);
+        KafkaProducer<String, byte[]> producer = new KafkaProducer<>(props);
 
-            long startTime = System.currentTimeMillis();
+        long startTime = System.currentTimeMillis();
```

Comment:
Following on Bill's nit, I configured my IDE to warn on every assignment that isn't final but could be. I only mention this extra fact because it might help you because there's a bunch of finalizable variables in here, and they would be easier to track down with IDE support.